### PR TITLE
edit EventDataQueryExpressionVisiter/Filter to accept value and key

### PR DESCRIPTION
**Pull request by austin007008**
edit EventDataQueryExpressionVisiter/Filter to accept value and key when they have already been parsed

### CODE REVIEW

File: warehouse/query-core/src/main/java/datawave/query/jexl/visitors/EventDataQueryExpressionVisitor.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
 import org.apache.commons.jexl2.parser.ASTNENode;
 import org.apache.commons.jexl2.parser.ASTNRNode;
 import org.apache.commons.jexl2.parser.JexlNode;
+import org.apache.commons.math3.analysis.function.Exp;
```
Comment (matthpeterson) [inline warehouse/query-core/src/main/java/datawave/query/jexl/visitors/EventDataQueryExpressionVisitor.java]: This import appears to be inadvertent.

File: warehouse/query-core/src/main/java/datawave/query/jexl/visitors/EventDataQueryExpressionVisitor.java [hunk 2]
```diff
@@ -155,21 +156,33 @@ public boolean apply(Key key) {
...
 
-        private boolean apply(Key key, boolean update) {
+        public boolean peek(Key key, String fieldName, String fieldValue) {
+            return apply(key, false, fieldName, fieldValue);
+        }
+
+        public boolean apply(Key key, boolean update) {
             final DatawaveKey datawaveKey = new DatawaveKey(key);
-            final String keyFieldName = JexlASTHelper.deconstructIdentifier(datawaveKey.getFieldName(), false);
+            final String FieldName = JexlASTHelper.deconstructIdentifier(datawaveKey.getFieldName(), false);
```
Comment (matthpeterson) [inline warehouse/query-core/src/main/java/datawave/query/jexl/visitors/EventDataQueryExpressionVisitor.java]: nit: fieldName would be a more consistent way to name this variable

**Comment by matthpeterson** [comment]
After the import removal, feel free to send me a direct message and I'll approve this MR.

File: warehouse/query-core/src/main/java/datawave/query/jexl/visitors/EventDataQueryExpressionVisitor.java (hunk 1 above)
Comment (austin007008) [reply warehouse/query-core/src/main/java/datawave/query/jexl/visitors/EventDataQueryExpressionVisitor.java]: removed

File: warehouse/query-core/src/main/java/datawave/query/jexl/visitors/EventDataQueryExpressionVisitor.java (hunk 2 above)
Comment (austin007008) [reply warehouse/query-core/src/main/java/datawave/query/jexl/visitors/EventDataQueryExpressionVisitor.java]: fixed

**Review by matthpeterson — DISMISSED** [review]

**Review by ivakegg — DISMISSED** [review]

**Pull request closed without merging**